docs/commands.md

```
@@ -0,0 +1,79 @@
+# Commands
+
+Terraform Branch Deploy supports a set of pull request commands for safe, auditable, and flexible infrastructure automation. Use these commands as PR comments to trigger actions.
+
+
+## Core Commands
+
+| Command Pattern                        | Description                                                      | Example Usage                |
+|----------------------------------------|------------------------------------------------------------------|------------------------------|
+| `.plan to <env>`                       | Preview changes for a specific environment (noop mode)            | `.plan to dev`               |
+| `.plan to <env> | <args>`              | Preview with extra Terraform CLI arguments                        | `.plan to dev | -var=debug=true` |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)